src/test/java/uk/gov/dhsc/htbhf/steps/CompleteFlowSteps.java

```diff
@@ -107,12 +108,10 @@ private void selectYesOnPregnancyPage() {
         areYouPregnantPage.clickContinue();
     }
 
-    private void enterTwoChildrensDatesOfBirth() {
+    private void enterOneChildsDateOfBirth() {
```

Comment:
This was also different to the web tests, we only normally add one child's details.